### ISSUE TITLE
Move MultiChooser#chooseNone to BaseChooser#chooseNone.

### ISSUE THREAD

**Issue by jgornick** [brian-mann/Backbone.Chooser#2]
Seeking a way to clear selection at the collection level (single and multiple). There is chooseNone in the MultiChooser, but an option doesn't exist for the SingleChooser.

Thoughts?

**Comment by brian-mann** [comment]
Good point.  I think perhaps instead of mixing these API's - and dealing with the consequences of SingleChooser having a chooseNone but not a chooseAll - I think I should make .unchoose() not require passing in a model.

Calling unchoose without any arguments should automatically unchoose the chosen model.  It get's a little hairy dealing with the options argument, but I can build the unchoose API to accept either a model to unchoose, or no model at all, and an options argument regardless.  

Would that suffice your use case?

**Comment by jgornick** [comment]
The method overloading option is cool.

However, personally I've tried to move away from that and create explicit methods. With that being said, a cleaner option I think would be to rename `chooseNone` to `clearChosen`. That naming pattern follows other methods like `getChosen`.
